_posts/2016-03-01-Docker-meets-Continuous-Deployment.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+name: docker-meets-continuous-deployment
+title: Docker meets Continuous Deployment
+date: 2016-03-01 00:00:00 +00:00
+author: Robert Firek
+image:
+    src: /assets/img/custom/blog/2016-03-01-docker-meets-continuous-deployment.png
+    attribution:
+        text: Track icon by Freepik (CC 3.0 BY)
+        href: http://www.freepik.com
+tags:
+- Continuous Deployment
+- Docker
+- TeamCity
+--- 
+
+About one year ago I had a first contact with Docker. This new kid on the block promised to relieve our poor computers from installation of all tools, languages, dependencies and operating systems. Isolated run environments emerged on developers computers.
+
+A conservative approach of my ops teammates didn't change my joy of using Docker. Despite many people describing Docker as tool writen by developers for development our industry found new ways of using images and containers. Images of our applications and services became units of deployment for tools like Kubernetes, Docker Swarm or Marathon.
+
+But how this images appear in these tools?
```

Comment:
But how do these images appear become available to these tools ? --- is this what you meant to say ?